### PR TITLE
Update Bracken to v2.6.1

### PR DESCRIPTION
**Pull request by martin-steinegger**
Update Bracken to v.2.6.1 
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by milot-mirdita** [comment]
@BiocondaBot please add label

**Comment by milot-mirdita** [comment]
@BiocondaBot please fetch artifacts

File: recipes/bracken/meta.yaml [hunk 1]
```diff
@@ -11,28 +11,30 @@ source:
...
     - {{ compiler('cxx') }}
     - llvm-openmp  # [osx]
+    - libgomp      # [linux]
   host:
     - python
   run:
     - python
     - kraken2
     - llvm-openmp  # [osx]
+    - libgomp      # [linux]
```
Comment (bgruening) [inline recipes/bracken/meta.yaml:30]: I think that is not needed, as we have a run_exports defined: https://github.com/conda-forge/ctng-compilers-feedstock/blob/9c214ed1ca8e16813e056243ef0c8bd65e054630/recipe/meta.yaml#L330

Comment (milot-mirdita) [reply recipes/bracken/meta.yaml:30]: I was copying this from MMseqs2 where @dpryan79 explicitly added both:
https://github.com/bioconda/bioconda-recipes/commit/a4001e18c0fdd964c9e9a60a1c47fb8da516a7b2#diff-2d85eb374c8cebc33fadbc998ec527e768d823ee8473bca28cfe16ac268f8c35

It was compiling correctly on Linux without the explicit dependency though. Should I remove it again?

Comment (bgruening) [reply recipes/bracken/meta.yaml:30]: But Devan has also only added it to the build section, not the run section, left?

Comment (milot-mirdita) [reply recipes/bracken/meta.yaml:30]: So is `llvm-openmp` in run also needed?

Comment (milot-mirdita) [reply recipes/bracken/meta.yaml]: I removed both lets see what happens

Comment (bgruening) [reply recipes/bracken/meta.yaml]: It seems to also have a run_export: https://github.com/conda-forge/openmp-feedstock/blob/master/recipe/meta.yaml#L31

**Review by bgruening — APPROVED** [review]

**Comment by milot-mirdita** [comment]
@BiocondaBot please merge

**Pull request merged**